recipes/magnum/all/conanfile.py

```diff
@@ -0,0 +1,710 @@
+        self._cmake.definitions["WITH_WAVAUDIOIMPORTER"] = self.options.with_wavaudioimporter
+
+        #### Command line utilities ####
+        self._cmake.definitions["WITH_GL_INFO"] = self.options.with_gl_info
+        self._cmake.definitions["WITH_AL_INFO"] = self.options.with_al_info
+        self._cmake.definitions["WITH_DISTANCEFIELDCONVERTER"] = self.options.get_safe("with_distancefieldconverter", False)
+        self._cmake.definitions["WITH_FONTCONVERTER"] = self.options.with_fontconverter
+        self._cmake.definitions["WITH_IMAGECONVERTER"] = self.options.with_imageconverter
+        self._cmake.definitions["WITH_SCENECONVERTER"] = self.options.with_sceneconverter
+
+        self._cmake.configure()
+        return self._cmake
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              'set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/modules/" ${CMAKE_MODULE_PATH})',
+                              "")
+        # Get rid of cmake_dependent_option, it can activate features when we try to disable them,
+        #   let the Conan user decide what to use and what not.
+        with open(os.path.join(self._source_subfolder, "CMakeLists.txt"), 'r+') as f:
+            text = f.read()
+            text = re.sub('cmake_dependent_option\(([0-9A-Z_]+) .*\)', r'option(\1 "Option \1 disabled by Conan" OFF)', text)
+            f.seek(0)
+            f.write(text)
+            f.truncate()
+
+        # GLFW naming
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "Magnum", "Platform", "CMakeLists.txt"),
+                              "find_package(GLFW)",
+                              "find_package(glfw3)")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "Magnum", "Platform", "CMakeLists.txt"),
+                              "GLFW_FOUND",
+                              "glfw3_FOUND")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "Magnum", "Platform", "CMakeLists.txt"),
+                              "GLFW::GLFW",
+                              "glfw::glfw")
+
+        # EGL naming
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "Magnum", "Platform", "CMakeLists.txt"),
+                              "find_package(EGL)",
+                              "find_package(egl_system)")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "Magnum", "Platform", "CMakeLists.txt"),
+                              "EGL_FOUND",
+                              "egl_system_FOUND")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "Magnum", "Platform", "CMakeLists.txt"),
+                              "EGL::EGL",
+                              "egl::egl")
```

Comment:
I need to understand why we are renaming to `egl_system` and, if it would be better to consume the `FindEGL.cmake` file provided by the library or CMake one.